### ISSUE TITLE
 font size in inspect (F12) windows is too small in high DPI screen

### ISSUE THREAD

**Issue by WanderMax** [goldendict/goldendict#985]
In high DPI screen like 4K, the font in inspect (F12) windows is too small to read.
Could you please offer any method to enlarge its size? or could you please improve this in future?
Thanks.
![inspect_small_font_size](https://user-images.githubusercontent.com/15909056/38086718-fea036a8-3387-11e8-95e7-5980b0adda60.png)

**Comment by Abs62** [comment]
If you use Qt5-based build, try standard keyboard combinations "Ctrl++", "Ctrl+-".

**Comment by WanderMax** [comment]
and can I set font size in CSS file to enlarge this size? is there any plan or method to auto fit the font based on windows scale factor?

**Comment by Abs62** [comment]
No. Web inspector is internal part of Qt WebKit, thing-in-itself.

**Comment by WanderMax** [comment]
@Abs62 can qt-style.css do work?

**Comment by Abs62** [comment]
No, qt-style.css affect GD interface only.

**Comment by WanderMax** [comment]
pity that I have to use the hotkey to enlarge font size every time when I use inspect feature

**Comment by WanderMax** [comment]
http://doc.qt.io/qt-5/highdpi.html
does it help?
